packages/react-scripts/template/src/app/app.test.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './app';
```

Comment:
Right, our starter had everything in lower case for file names but the standard is indeed capital and I can see that from their create-react-app that it is capital as well.